src/migrations/20241115203616-add-post-processing-to-import.js

```diff
@@ -0,0 +1,42 @@
+const { GROUP_SHARED_WITH } = require('@ttahub/common');
+const {
+  prepMigration,
+  removeTables,
+} = require('../lib/migration');
+const { GROUP_COLLABORATORS } = require('../constants');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+
+      // Add column postProcessingActions to table Imports of JSONB type.
+      await queryInterface.addColumn(
+        'Imports',
+        'postProcessingActions',
+        {
+          type: Sequelize.JSONB,
+          allowNull: true,
+        },
+        { transaction },
+      );
+
+      // Update Imports set the postProcessingActions column to the object.
+      await queryInterface.sequelize.query(/* sql */`
+            UPDATE "Imports"
+            SET "postProcessingActions" = '{"name": "Monitoring Goal CRON job", "function": "createMonitoringGoals"}'
```

Comment:
In `postProcess.test.js`, all of the mock records that are created define `postProcessingActions` as an array of objects, shaped like:

```ts
[
  { name: string; function: string; }
]
```

That's good. But...

Your migration above defines `postProcessingActions` as an object, not an array. So, while the tests may pass because the mocks provide good data, it seems to me that the below code (taken from `postProcess.ts`) will fail because objects do not have a `.map` method on their prototype:

```ts
    // Loop and execute post processing actions.
    await Promise.all(importRecord.postProcessingActions.map(async (action) => {
      switch (action.function) {
```

Reply:
I wrapped the postProcess value in [] array.